docs/ops/image/NV12toBGR_8.md

```diff
@@ -0,0 +1,82 @@
+## NV12toBGR <a name="NV12toBGR"></a> {#openvino_docs_ops_image_NV12toBGR_8}
+
+**Versioned name**: *NV12toBGR-8*
+
+**Category**: *Image processing*
+
+**Short description**: *NV12toBGR* performs image conversion from NV12 format to BGR.
+
+**Detailed description**:
+
+Similar to *NV12toRGB* but output channels for each pixel are reversed so that: first channel is `blue`, second one is `green`, last one is `red`.  See detailed conversion formulas at [NV12toRGB](NV12toRGB_8.md)
+
+**Inputs:**
+
+Same as specified for [NV12toRGB](NV12toRGB_8.md) operation.
+
+**Outputs:**
+
+* **1**: A tensor of type *T* representing converted image in BGR format. Dimensions:
+  * `N` - batch dimension
+  * `H` - height dimension is same as image height
+  * `W` - width dimension is same as image width
+  * `C` - channels dimension equals to 3. First channel is Blue, second one is Green, last one is Red
+
+**Types:**
+
+* *T*: `uint8` or `float32` type.
```

Comment:
Shouldn't `float32` be replaced with `any supported floating-point type`?

Reply:
@ilya-lavrenov can you comment on this? Technically any type except 'bool', 'int4', 'int8' is fine to store range 0-255

Reply:
I'm OK with it, but will plugins support it?

@jane-intel what is the policy for such cases?

Reply:
*T*: `uint8` or any supported floating-point type.

Reply:
Thanks, corrected